Conceptpower+Spring/src/main/java/edu/asu/conceptpower/rest/ConceptIDLookup.java

```diff
@@ -88,7 +88,7 @@
 
         XMLConceptMessage msg = messageFactory.createXMLConceptMessage();
 
-        if (entry != null) {
+        if (entry != null && !entry.isDeleted()) {
```

Comment:
@jdamerow  I am not able to understand. This method will be called only through rest interface when the user searches a concept based on id. I am not sure how to differentiate between lookup and searching.